cli/ss2

```diff
@@ -578,7 +584,7 @@ def run(args=None):
     with DiagSocket() as ds:
         ds.bind()
         for p in protocols:
-            p(ds, args, _user_ctxt_map)
+                p(ds, args, _user_ctxt_map)
```

Comment:
lapsus!

Reply:
indeed

Reply:
this looks over-indented. You use 4-wide everywhere, but 8 here

Reply:
fixed